.github/workflows/test.yml

```diff
@@ -210,6 +207,12 @@ jobs:
               shell: bash -l {0}
               working-directory: ${{ matrix.example_directories }}
               run: source venv/bin/activate && pip install -r requirements.txt
+            - if: ${{ needs.release-candidate-deploy.outputs.should_run_tests }}
+              working-directory: ${{ matrix.example_directories }}
+              run: source venv/bin/activate && python -m kybra install-dfx-extension
```

Comment:
Same question for me as in Azle, should we automatically install this? I'm waiting on some guidance for this, we might be able to do this automatically soon...we just need to be able to run dfx start in the same directory as a dfx.json file with an unsupported extension